plugins/chat/assets/javascripts/discourse/models/chat-thread-preview.js

```diff
@@ -1,18 +1,34 @@
 import { tracked } from "@glimmer/tracking";
+import { TrackedArray } from "@ember-compat/tracked-built-ins";
 
 export default class ChatThreadPreview {
   static create(args = {}) {
     return new ChatThreadPreview(args);
   }
 
+  @tracked replyCount;
   @tracked lastReplyId;
   @tracked lastReplyCreatedAt;
   @tracked lastReplyExcerpt;
+  @tracked lastReplyUser;
+  @tracked participantCount;
+  @tracked participantUsers = [];
```

Comment:
probably want to to not assign it an empty array here, it's either not correct as we want tracked or just useless as we overwrite it right away